cwms_radar_api/src/main/java/cwms/radar/api/ClobController.java

```diff
@@ -124,14 +125,23 @@ public void getAll(Context ctx) {
             requestResultSize.update(result.length());
 
         } catch ( FormattingException fe ){
-            logger.log(Level.SEVERE,"failed to format data",fe);
+            RadarError re = null;
+
             if( fe.getCause() instanceof IOException ){
+                re = new RadarError("failed to format data", null );
                 ctx.status(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
                 ctx.result("server error");
             } else {
-                ctx.status(HttpServletResponse.SC_BAD_REQUEST);
-                ctx.result("Invalid Format Options");
+                re = new RadarError("failed to format data", new HashMap<String,String>(){
+                    {
+                        put("cause","Invalid format or format options");
+                    }
+                });
+                ctx.status(HttpServletResponse.SC_NOT_IMPLEMENTED);
+
             }
+            logger.log(Level.SEVERE,String.format("%s: failed to format data",re.getIncidentIdentifier()),fe);
```

Comment:
Maybe just get the message from the RadarError ctor instead?
Or even just log it inside the ctor instead of a separate statement? Though, if there's a reason to construct a RadarError class that doesn't also need to be logged, then that might not work.

Reply:
I think I just missed tweaking that one. I made all of the other instances re.toString() for the log message. And put the formatting of the incident ID and message in that.

Reply:
> I think I just missed tweaking that one. I made all of the other instances re.toString() for the log message. And put the formatting of the incident ID and message in that.

Oh, in that case, there's a few more below you missed.

Reply:
Looks like an improvement to me.